src/mme-app/utils/mmeDnsMsgUtils.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+* Copyright 2020-present, Infosys Ltd.
+*
+* SPDX-License-Identifier: Apache-2.0
+*/
+
+#include "utils/mmeDnsMsgUtils.h"
+
+#include <controlBlock.h>
+#include <contextManager/dataBlocks.h>
+#include <mme_app.h>
+#include "epc/epcdns.h"
+
+using namespace mme;
+
+extern mme_config_t *mme_cfg;
+
+
+void MmeDnsMsgUtils::Ipv4_gateway_ip( EPCDNS::StringVector &ipv4_ip, unsigned long * sgw_ip)
+{
+        EPCDNS::StringVector::const_iterator it = ipv4_ip.begin();
+        std::cout<<*it <<"\n";
+        char * result = (char*)ipv4_ip[0].c_str();
+        std::cout << result << "\n";
+        unsigned long tmp = htonl(inet_addr(result));
+        memcpy(sgw_ip,&tmp, sizeof(unsigned long));
+        std::cout << *sgw_ip << "\n";
+}
+void MmeDnsMsgUtils::Extract_IPs( EPCDNS::NodeSelectorResultList &at, unsigned long  *sgw_ip)
+{
+        EPCDNS::NodeSelectorResultList::const_iterator it = at.begin();
+        Ipv4_gateway_ip((*it)->getIPv4Hosts(), sgw_ip);
+}
+
+void MmeDnsMsgUtils::SetDNSConfiguration()
+{
+        DNS::Cache::setRefreshConcurrent( mme_cfg->dns_config.concurrent);
+        DNS::Cache::setRefreshPercent( mme_cfg->dns_config.percentage);
+        DNS::Cache::setRefreshInterval( mme_cfg->dns_config.interval_seconds );
+	std::cout<< "address of dns server" << mme_cfg->dns_config.dns1_ip;
```

Comment:
remove unwanted print

Reply:
this is added purposely so that we it will be easy to capture issue if there is some issue with DNS server address and in case of multiple DNS (for future), it will capture what DNS server we are using rather than checking conf file again and again.
If need to remove can be done. Kindly confirm.